### DISCUSSION TITLE
Problem in Github website

### DISCUSSION THREAD

**Discussion by tymsai** [github/india#114] [GitHub Platform]
There is an mis alignment in the css and html  github pull request site.
where can I report it ???

**Comment by TarunavBA** [comment]
@tymsai can you share a picture of it?

**Comment by tymsai** [comment]
its resolved :-)

[the remark this answers is not in the document]
**Reply by Aditya-Bhate** [reply]
@tymsai Github is working fine at my end

**Comment by Aditya-Bhate** [comment]
Github is working fine at my end

**Comment by FRTMentor** [comment]
Please reach out the GitHub support team.
